hopbox/hopbox_app/templates/option.html

```diff
@@ -13,7 +13,7 @@
     <nav class="navbar navbar-expand-lg navbar-white  bg-white mb-5" style="border-bottom: 3px solid #91B957;">
         <div class="container-fluid d-flex justify-content-evenly">
             <a class="navbar-brand ms-2 fw-bold active" href="/userdashboard">
-                <img src="{% static 'HopBox-logo.png' %}" alt="logo_placeholder" width="340" height="155"
+                <img src="static/images/HopBox-logo.png" alt="logo_placeholder" width="340" height="155"
```

Comment:
same thing as below, I think we need to use the django templating engine format for static file rendering